admin/billing/payment/stripe.go

```diff
@@ -80,11 +93,24 @@ func (s *Stripe) FindCustomerForOrg(ctx context.Context, organization *database.
 			it := customer.ListPaymentMethods(&stripe.CustomerListPaymentMethodsParams{
 				Customer: stripe.String(c.ID),
 			})
+			hasPaymentMethod := false
+			var isCardValid *bool
+			// very basic check if the customer has a payment method and if it's a card then is not expired
+			for it.Next() {
+				hasPaymentMethod = true
+				pm := it.PaymentMethod()
+				isCardValid = new(bool)
+				if pm.Type == stripe.PaymentMethodTypeCard {
+					*isCardValid = int(pm.Card.ExpYear) > time.Now().Year() || (int(pm.Card.ExpYear) == time.Now().Year() && int(pm.Card.ExpMonth) >= int(time.Now().Month()))
+				}
+			}
```

Comment:
Since this is duplicated twice and non-trivial, can we pull it into a util func?